concepts/05 UI Components/SpeechToText/00 Overview/05 Elements.md

```diff
@@ -0,0 +1,6 @@
+<img src="/images/SpeechToText/speechtotext-elements.png" alt="SpeechToText elements"/>
+
+1. [Start Icon](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#startIcon)
```

Comment:
We can use 1 for both (1) and (3)
and 2 for both (2) and (4) to simplify this list to 2 items

Reply:
In this case (3) and (4)'s red dots better to set closer to the text because currently it is not obvious